BadHabbit/BadHabbit/UserStories/SingnInAppleScreen/SignInAppleViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SignInAppleViewController.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 14.07.23.
+//
+
+import UIKit
+import SnapKit
+import AuthenticationServices
+
+class SignInAppleViewController: UIViewController {
+    private var authorizationButton: ASAuthorizationAppleIDButton!
+    private var imageLogo: UIImageView!
+    private var imageMountains: UIImageView!
+    private var titleLabel: UILabel!
```

Comment:
To avoid ugly and unsafe force unwraps values lets make lazy vars

Reply:
changed